packages/rulesets/generated/spectral/az-arm.js

```diff
@@ -3623,7 +3613,6 @@ const ruleset = {
             description: "This is a rule introduced to validate if provisioningState property is set to readOnly or not.",
             message: "{{error}}",
             severity: "warn",
```

Comment:
>warn [](http://example.com/codeflow?start=23&length=4)

error

Reply:
Not sure why I has this as warn, weird.
Updated